pkg/logfilter/logfilter.go

```diff
@@ -188,9 +214,14 @@ func FilterOutput(logFilters []*LogFilter, output string) string {
 					continue
 				}
 			}
-			if filter.summaryText == "" {
+			if filter.enhancedThreshold > 0 && int64(math.Abs(filteredVal)) > filter.enhancedThreshold {
+				if len(filter.offsets) > 0 {
```

Comment:
should this be greater than 1? It doesn't seem like there is much point if the only offset is the one we're about to output.

Reply:
If we trip based on the threshold, we never append the one that we're about to print.  So this would be the summary of all previous ones before the one that we're about to print.